config-model/src/main/java/com/yahoo/vespa/model/content/cluster/DomResourceLimitsBuilder.java

```diff
@@ -1,20 +1,28 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.model.content.cluster;
 
+import com.yahoo.config.application.api.DeployLogger;
 import com.yahoo.vespa.model.builder.xml.dom.ModelElement;
 import com.yahoo.vespa.model.content.ResourceLimits;
 
+import java.util.logging.Level;
+
 /**
  * Builder for feed block resource limits.
  *
  * @author geirst
  */
 public class DomResourceLimitsBuilder {
 
-    public static ResourceLimits.Builder createBuilder(ModelElement contentXml) {
+    public static ResourceLimits.Builder createBuilder(ModelElement contentXml, boolean hostedVespa, DeployLogger deployLogger) {
         ResourceLimits.Builder builder = new ResourceLimits.Builder();
         ModelElement resourceLimits = contentXml.child("resource-limits");
-        if (resourceLimits == null) {
+        if (resourceLimits == null) { return builder; }
+
+        if (hostedVespa) {
+            deployLogger.log(Level.WARNING, "Element " + resourceLimits + " is not allowed, default limits will be used");
```

Comment:
Log with `.logApplicationPackage()` so the tenant gets a notification in console?

Reply:
Yes, thanks, done in https://github.com/vespa-engine/vespa/pull/17948